cmd/habitatctl/commands/helpers.go

```diff
@@ -3,16 +3,16 @@ package commands
 import (
 	"fmt"
 
-	"github.com/eagraf/habitat/cmd/habitatctl/client"
+	client "github.com/eagraf/habitat/pkg/habitat_client"
 	"github.com/eagraf/habitat/structs/ctl"
 	"github.com/spf13/viper"
 )
 
-func sendRequest(command string, args []string) {
+func SendRequest(command string, args []string) (*ctl.Response, error) {
```

Comment:
I think that this helper method should be included in the pkg/habitat_client, since this is the method that all the other usages in the community app are referring to